sdks/python/apache_beam/runners/worker/statecache.py

```diff
@@ -231,7 +231,7 @@ class StateCache(object):
   """
   def __init__(self, max_weight):
     # type: (int) -> None
-    _LOGGER.info('Creating state cache with size %s', max_weight)
+    _LOGGER.debug('Creating state cache with size %s', max_weight)
```

Comment:
Having this line can be useful when running on production runners, i'd prefer to keep it.

Reply:
Reverted this